os/sdl/gl_screen.h

```diff
@@ -0,0 +1,39 @@
+//
+// os/sdl/gl_screen.h: SDL screen definitions.
+//
+// CEN64: Cycle-Accurate Nintendo 64 Emulator.
+// Copyright (C) 2015, Tyler J. Stachecki.
+//
+// This file is subject to the terms and conditions defined in
+// 'LICENSE', which is part of this source code package.
+//
+
+#ifndef CEN64_OS_SDL_GL_SCREEN
+#define CEN64_OS_SDL_GL_SCREEN
+#include "gl_common.h"
+#include <stddef.h>
+
+#define CEN64_GL_SCREEN_BAD (NULL)
+typedef int cen64_gl_screen;
+
+//
+// Creates a cen64_gl_screen object on a given cen64_gl_display.
+//
+// 'which' specifies which screen on a display to use (from 0 to
+// num_screens - 1). If 'which' is less than zero, it is treated as
+// a don't care.
+//
+// On error, CEN64_GL_SCREEN_BAD is returned.
+//
+static inline cen64_gl_screen cen64_gl_screen_create(
+  cen64_gl_display display, int which) {
+    return 0x1234;
```

Comment:
`0x1234` is a bit unclear at a first glance, change to 0, put a comment explaining that it's not used by the SDL backends, and change CEN64_GL_SCREEN_BAD to (-1) so that it's most similar to the X11 backend.